### DISCUSSION TITLE
用GitHub Actions打包的应用能打开devtools吗

### DISCUSSION THREAD

**Discussion by wadejs** [tw93/Pake#683] [Q&A]
用GitHub Actions打包的应用能打开devtools吗

**Comment by tw93** [comment]
哈哈，现在不支持了，不过我感觉必要性不大，假如都需要devtools，那很建议直接fork Pake，用高级模式开发会方便很多，因为不仅仅是修改这里面的东西，或者你看看这个使用场景是啥，也可以一起讨论，场景多，我们可以一起看看要不要加上去
